native/acter/src/api/room.rs

```diff
@@ -753,24 +753,24 @@ impl Room {
 
     pub async fn set_name(&self, name: String) -> Result<OwnedEventId> {
         if !self.is_joined() {
-            bail!("Can't set name to a room we are not in");
+            bail!("Unable to set name to a room we are not in");
         }
         let room = self.room.clone();
 
         let my_id = room
             .client()
             .user_id()
-            .context("User not found")?
+            .context("UserId not found")?
```

Comment:
This isn't more clear. `client.user_id()` should probably always have a context saying `context("You must be logged in to do that")`...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1252/commits/b8628e84f7cc4497b318ff0e28569c4ff937d00f